platform/admin/docs/src/main/resources/ManContents.adoc

```diff
@@ -1,99 +1,83 @@
 
-The administrative application enhances administrative capabilities when installing and managing DDF. It contains various services and interfaces that allow administrators more control over their systems.
+It contains various services and interfaces that allow administrators more control over their systems and enhances administrative capabilities when installing and managing {branding}.
 
-The Admin application contains an application service that handles all operations that are performed on applications. This includes adding, removing, starting, stopping, and showing status.
+The Admin application contains an application service that handles all operations that are performed on applications.
+This includes adding, removing, starting, stopping, and showing status.
 
-=== Administrative User Interface
-
-The Admin UI is the centralized location for administering the system. The Admin UI allows an administrator to install and remove selected applications and their dependencies and access configuration pages to configure and tailor system services and properties
+=== Installing {branding} Admin
 
-=== Modules
+The {branding} Admin application is installed by default with a standard installation.
 
-The Admin UI is a modular system that can be expanded with additional modules as necessary. DDF comes with the Configurations module and the Installation modules. However, new modules can be added, and each module is presented in its own tab of the Admin UI. More information on modules, including the ones that come with DDF, is available on the Modules page.
+=== Configuring {branding} Admin
 
-== Prerequisites
-Before the DDF Admin application can be installed:
+[cols="1,1,3" options="header"]
+|===
 
-* the DDF Kernel must be running
-* the DDF Platform Application must be installed
-* the DDF Catalog Application must be installed
+|Configuration
+|ID
+|Description
 
-== Install Applications
+|Admin UI Configuration
+|`org.codice.admin.ui.configuration`
+|Customization options for Admin UI elements.
 
-* Before installing a {branding} application, verify that its prerequisites have been met. 
-* Copy the {branding} application's KAR file to the `<INSTALL_DIRECTORY>/deploy` directory.
+|Configuration Admin JMX Bridge
+|`ddf.ui.admin.api`
+|List of Configuration Filters
 
-== Verify
+==== Defining Platform Settings
 
-. Verify the appropriate features for the DDF application have been installed using the
-`features:list` command to view the KAR file's features.
+The platform settings can be configured by a file located in `DDF_HOME/etc/ddf.platform.config.cfg`
 
-. Verify that the bundles within the installed features are in an active state.
+The settings in this file can be changed at any time during the {branding} lifecycle (before installation, after, or during run-time) and will immediately update the corresponding properties for the platform global configuration.
```

Comment:
Again no platform global configuration anymore. Changes to the system.properites file requires a system restart.